roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
+
+# namespace
+- name: Add prometheus project
+  oc_project:
+    state: "{{ state }}"
+    name: "{{ openshift_prometheus_namespace }}"
+    node_selector: "{{ openshift_prometheus_node_selector | regex_replace(':','=') | regex_replace('{', '[') | regex_replace('}', ']') }}"
```

Comment:
@joelddiaz I added the node selector field to oc_project.
It seems to require specific formating 'region=infra' and not support "region:infra" filter that is supported by deployment object, so I had to hack it a bit.
This works on my cluster. could you check on yours?